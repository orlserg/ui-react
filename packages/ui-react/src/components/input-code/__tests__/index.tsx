import React from 'react';
import { render } from '@testing-library/react';

import { InputCode } from '..';

test('Default', () => {
    const { asFragment } = render(
        <InputCode />
    );

    expect(asFragment()).toMatchSnapshot();
});

test('disabled', () => {
    const { asFragment } = render(
        <InputCode
            disabled={true}
        />
    );

    expect(asFragment()).toMatchSnapshot();
});
