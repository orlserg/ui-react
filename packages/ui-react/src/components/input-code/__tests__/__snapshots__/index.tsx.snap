// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default 1`] = `
<DocumentFragment>
  <div
    class="input-code"
  >
    <input
      autocomplete="off"
      class="input-code__input"
      max="9"
      maxlength="6"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;

exports[`disabled 1`] = `
<DocumentFragment>
  <div
    class="input-code"
  >
    <input
      autocomplete="off"
      class="input-code__input"
      disabled=""
      max="9"
      maxlength="6"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      disabled=""
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      disabled=""
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      disabled=""
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      disabled=""
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
    <input
      autocomplete="off"
      class="input-code__input"
      disabled=""
      max="9"
      maxlength="1"
      min="0"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
